CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h

```diff
@@ -227,6 +227,8 @@ extern void _CFDataInit(CFMutableDataRef memory, CFOptionFlags flags, CFIndex ca
 extern int32_t _CF_SOCK_STREAM();
 
 extern CFStringRef CFCopySystemVersionString(void);
+extern CFDictionaryRef _CFCopySystemVersionDictionary(void);
+extern const CFStringRef _kCFSystemVersionProductVersionKey;
```

Comment:
You're partly right. `CFPriv.h` declares `_kCFSystemVersionProductVersionKey` but not `_CFCopySystemVersionDictionary`, so I just removed the key from here.